backend/database/schema.prisma

```diff
@@ -24,7 +24,7 @@ model Collection {
     description             String?
     embedder_config         Json
     // Collection can have multiple data sources
-    associated_data_sources Json?
+    associated_data_sources Json    @default("{}")
```

Comment:
Will primsa auto migrate older NULL entries?

Reply:
There are no null values in the DB. It's only during collection creation API is split first into creating the collection and attaching the DB. But yes if there are prisma should be able to do.